cime/config/e3sm/machines/config_machines.xml

```diff
@@ -593,49 +344,45 @@
       <env name="OMP_PROC_BIND">spread</env>
       <env name="OMP_PLACES">threads</env>
       <env name="HDF5_USE_FILE_LOCKING">FALSE</env>
+
+      <env name="PERL5LIB">/project/projectdirs/acme/perl5/lib/perl5/x86_64-linux-thread-multi</env>
     </environment_variables>
 
     <environment_variables mpilib="mpt">
       <env name="MPICH_GNI_DYNAMIC_CONN">disabled</env>
     </environment_variables>
-    <environment_variables mpilib="impi">
-      <env name="I_MPI_FABRICS">ofi</env>
-      <env name="I_MPI_OFI_PROVIDER">gni</env>
-      <env name="I_MPI_PRINT_VERSION">yes</env>
-      <env name="I_MPI_OFI_LIBRARY">/global/common/cori/software/libfabric/1.6.1/gnu/lib/libfabric.so</env>
-      <env name="I_MPI_PMI_LIBRARY">/usr/lib64/slurmpmi/libpmi.so</env>
-    </environment_variables>
     <environment_variables compiler="intel">
       <env name="FORT_BUFFERED">yes</env>
       <env name="MPICH_MEMORY_REPORT">1</env>
     </environment_variables>
-    <environment_variables compiler="intel19">
-      <env name="MPICH_MEMORY_REPORT">1</env>
-    </environment_variables>
   </machine>
 
   <!-- Skylake nodes of Stampede2 at TACC -->
   <machine MACH="stampede2">
-    <DESC>Stampede2. Intel skylake nodes at TACC. 48 cores per node, batch system is SLURM</DESC>
+    <DESC>Stampede2. Intel skylake nodes at TACC.  48 cores per node, batch system is SLURM</DESC>
```

Comment:
sorry, these changes and a few more below for machines that have nothing to do with NERSC, came about while resolving the conflicts of the cherry-pick.